settings.py

```diff
@@ -1069,6 +1069,8 @@ def _pkce_required(client_id):
     },
 }
 
+ES_CASE_SEARCH_INDEX_NAME = "case_search_2022-10-14"
```

Comment:
I think this change should be applied to the default value here: https://github.com/dimagi/commcare-hq/blob/0bc9a3e3d30082006130038db40adb010ab8590c/corehq/apps/es/case_search.py#L142